docs/rsdl/rapid-pro-rsdl-semantics.mdx

```diff
@@ -147,6 +147,82 @@ type Name {
 </TabItem>
 </Tabs>
 
+### Abstract Structured Types
```

Comment:
Would like to make this an example based on the Jetsons schema. Maybe something like:
```
abstract type Vehicle {
   manufacturer : string
   model: string
}
```

```
type FlyingCar extends Vehicle {
   capacity: integer
}
```